pandas/core/groupby/groupby.py

```diff
@@ -486,6 +486,12 @@ def __repr__(self) -> str:
         # TODO: Better repr for GroupBy object
         return object.__repr__(self)
 
+    @property
+    def dropna(self) -> bool:
+        if self._dropna is lib.no_default:
+            return True
+        return self._dropna
```

Comment:
I know the implementation is trivial, but this is redundant with `Grouper`. I'm not sure we can get around it while still being a class property, but should the default value be referenced as a constant defined just once?